gustaf/show.py

```diff
@@ -43,6 +50,19 @@ def __call__(self, *args, **kwargs):
 sys.modules[__name__].__class__ = _CallableShowDotPy
 
 
+def is_ipython() -> bool:
```

Comment:
```suggestion
def is_ipython():
```
great, but in docs please :D